### PR TITLE
PostRevisions: Add translation contexts to "minor" and "cancel"

### PR DESCRIPTION
**Pull request by spen**
### Summary

This PR aims to address the ambiguous translations pointed out here: https://github.com/Automattic/wp-calypso/pull/19197#pullrequestreview-76354363 by adding context parameters to each case.

### Testing
No testing necessary, please just check that each is improved and that the messages given are useful :)

### CODE REVIEW

File: client/post-editor/editor-revisions/dialog.jsx [hunk 1]
```diff
@@ -79,7 +79,11 @@ class PostRevisionsDialog extends PureComponent {
 	dialogButtons = () => {
 		const { postId, revision, siteId, translate } = this.props;
 		return [
-			{ action: 'cancel', compact: true, label: translate( 'Cancel' ) },
+			{
+				action: 'cancel',
+				compact: true,
+				label: translate( 'Cancel', { context: 'modal button' } ),
```
Comment (yoavf) [inline client/post-editor/editor-revisions/dialog.jsx]: That was is not necessary - "Load" was the 2nd string I pointed out.

File: client/post-editor/editor-revisions-list/item.jsx [hunk 2]
```diff
@@ -70,7 +70,9 @@ class EditorRevisionsListItem extends PureComponent {
 
 					{ added === 0 &&
 						removed === 0 && (
-							<span className="editor-revisions-list__minor-changes">{ translate( 'minor' ) }</span>
+							<span className="editor-revisions-list__minor-changes">
+								{ translate( 'minor', { context: 'post revisions: minor changes' } ) }
```
Comment (yoavf) [inline client/post-editor/editor-revisions-list/item.jsx]: 👍

Comment (a8ci18n) [inline client/post-editor/editor-revisions-list/item.jsx]: Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Minor: ' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

File: client/post-editor/editor-revisions/dialog.jsx (hunk 1 above)
Comment (a8ci18n) [inline client/post-editor/editor-revisions/dialog.jsx]: :disappointed: 29 existing translations will be lost with this change.
If the string must change, the following option has already been translated **28** times:
`translate( 'Cancel', { context: 'Customizer cancel element editing button'} )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/post-revisions/add-context-to-ambiguous-translations)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (spen) [reply client/post-editor/editor-revisions/dialog.jsx]: Oops good spot! Thanks!

File: client/post-editor/editor-revisions/dialog.jsx [hunk 3]
```diff
@@ -85,7 +85,7 @@ class PostRevisionsDialog extends PureComponent {
 				compact: true,
 				disabled: ! ( revision && postId && siteId ),
 				isPrimary: true,
-				label: translate( 'Load' ),
+				label: translate( 'Load', { context: 'Load revision in editor' } ),
```
Comment (a8ci18n) [inline client/post-editor/editor-revisions/dialog.jsx]: :disappointed: 1 existing translations will be lost with this change.

File: client/post-editor/editor-revisions/dialog.jsx (hunk 1 above)
Comment (spen) [reply client/post-editor/editor-revisions/dialog.jsx]: Thanks @yoavf I've removed this change and now added the context to 'Load' :)

**Pull request merged**